var/spack/repos/builtin/packages/py-xlrd/package.py

```diff
@@ -14,3 +14,6 @@ class PyXlrd(PythonPackage):
     url      = "https://pypi.io/packages/source/x/xlrd/xlrd-0.9.4.tar.gz"
 
     version('0.9.4', sha256='8e8d3359f39541a6ff937f4030db54864836a06e42988c452db5b6b86d29ea72')
+    version('1.0.0', sha256='0ff87dd5d50425084f7219cb6f86bb3eb5aa29063f53d50bf270ed007e941069')
```

Comment:
`nitpick` : put latest version at the top

Reply:
Thanks, done in the next commit.